packages/components/src/draggable/index.native.js

```diff
@@ -77,25 +77,39 @@ const Draggable = ( { children, onDragEnd, onDragOver, onDragStart } ) => {
 		}
 	);
 
+	function getFirstTouchEvent( event ) {
+		'worklet';
+
+		const sortedEventsById = event.allTouches.sort(
+			( a, b ) => b.id + a.id
```

Comment:
It sorts the touches by descending order because we can't rely on the ID `0` on the Android simulator for example, the first touch ID is `2`. This way it always gets the first one regardless if the IDs are `0` or `2`

Reply:
Not sure if the library guarantees the lowest id is the first touch 🤔 . Maybe we could just keep a reference of the first id when `onTouchesDown` is triggered to know this, wdty?